Sources/PackageModel/IdentityResolver.swift

```diff
@@ -11,7 +11,6 @@
 //===----------------------------------------------------------------------===//
 
 import Foundation
-import PackageModel
```

Comment:
Since `ContextModel` is part of `PackageLoading` and `PackageDescription` via a symlink, we need to import `Foundation` implementation-only in both modules. This is incompatible with `IdentityResolver` since it uses `Foundation.URL` as part of its public interface. To resolve this, I moved `IdentityResolver` into `PackageModel` here.